### PR TITLE
Update initial readme for the repository

### LINKED ISSUE #6: Chore: Drafting README Content for the repository
# Description
This issue is dedicated to planning the content and structure of the repository's README file.
Share your ideas, suggestions, and feedback on what should be included in the README.
Let's collaborate to create a clear and informative README that shows clear understanding of the repository.

## Proposed Sections:
- [x] Title and Description
- [x] Installation
- [x] Usage

## Deadline:
30th April 2024

### PR DESCRIPTION
**Pull request by kibongos40**
# Pull Request Description:

This PR updates the README for clarity. Here's a breakdown of the changes:

- Updated about info for the repo and languages used.
- Updated installation instructions to run the app.
- Clarified usage examples for better understanding.

## Reason for Changes:

The current README was not updated and lacked clarity for the repository. These updates aim to make it easier for new users to get started with the project and know more about the project.

## Testing:

I will test the installation instructions on a clean environment to ensure accuracy after the main codebase is initialised.

## Notes for Reviewers:

Please let me know if there are any additional changes or improvements you'd like to see in the README. I'm open to feedback and suggestions to make the documentation more helpful for users.

### CODE REVIEW

**Review by chrysloi — APPROVED** [review]

**Review by fredshema — APPROVED** [review]

**Review by fredshema — APPROVED** [review]

**Review by Adadie — COMMENTED** [review]
Why do we have these settings?

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (Adadie) [inline .vscode/settings.json:1]: Why do we have these settings?

**Review by 911samuel — APPROVED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (Adadie) [reply .vscode/settings.json:1]: cc @kibongos40

**Review by Adadie — CHANGES_REQUESTED** [review]

**Review by twizerim — APPROVED** [review]

**Comment by mukunzidd** [comment]
@kibongos40 your PR targets `main` you should target `dev` instead. I'm closing this so go ahead and open a new one pointing to the proper branch.

**Pull request closed without merging**